projects/canopy/src/lib/forms/radio/docs/segment/guide.stories.mdx

```diff
@@ -0,0 +1,113 @@
+import { Meta, Description } from '@storybook/addon-docs';
+import feedback from '../../../../docs/common/feedback.md';
+
+<Meta title="Components/Form/Segment/Guide" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />
+
+# Segment
+
+<p class="standfirst">The segment control can be used on forms where the user is required to make a conscious decision between several choices (min 2, max 5) and vertical space on desktop is limited.</p>
+
+![example](docs/segment/example.png)
+
+## Usage
+
+Use a segment control when users need to make one selection from multiple options and where vertical space is limited. 
+
+It's also intended as an alternative to radio buttons when the user needs to make a conscious choice i.e. unlike a radio button which always has one option selected by default.
+
+Import the radio module into your module:
```

Comment:
Could we add this line before the import sentence please? `The segment component is a variant of the radio component.`